charts/datasaur/templates/main/llm-lab-frontend-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.ingress.llmLabs.host }}
```

Comment:
this file also should be renamed to llm-labs-frontend-deployment.yaml

Reply:
done @tonoman3g